tests/Fixtures/app/console

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env php
```

Comment:
Yeah makes sense for a Symfony project, but not for a Symfony application used for test purposes like here does it? This `console` is used only by us developers, not the package consumer, so should also be made easy to be used.